src/main/java/net/rcarz/jiraclient/Project.java

```diff
@@ -170,5 +175,26 @@ public String getAssigneeType() {
     public Map<String, String> getRoles() {
         return roles;
     }
+
+    public List<User> searchAssignableUsers() throws JiraException {
+        return searchAssignableUsers(0, 50);
+    }
+
+    public List<User> searchAssignableUsers(int startIndex, int count) throws JiraException {
+        try {
+            Map<String,String> param = new HashMap<>();
+            param.put("project", key);
+            param.put("startAt", Integer.toString(startIndex));
+            param.put("maxResults", Integer.toString(count));
+
+            URI uri = restclient.buildURI(Resource.getBaseUri() + "user/assignable/search", param);
+            JSONArray response = (JSONArray) restclient.get(uri);
+            List<User> users = (List<User>)response.stream().map(o -> new User(restclient, (JSONObject) o)).collect(Collectors.toList());
```

Comment:
Although Java 7 is EOL, it's probably still used a lot in the wild. Can you replace the lambda?